docs/docs-content/clusters/public-cloud/azure/create-azure-cluster.md

```diff
@@ -56,8 +56,16 @@ to create an IaaS Kubernetes cluster in Azure that Palette manages.
   - Managed Disks
   - Virtual Network Address Translation (NAT) Gateway
 
-- To enable the `fullyPrivateAddressing` parameter, you need a self-hosted Private Cloud Gateway (PCG) deployed in
-  Azure. For more information on deploying PCGs, refer to [Private Cloud Gateway](../../pcg/pcg.md).
+- To enable the `fullyPrivateAddressing` parameter and use a Private API Server load balancer, you need a self-hosted
+  Private Cloud Gateway (PCG) deployed in Azure. Ensure the Azure cloud account selected is connected to a PCG. For more
+  information on deploying PCGs, refer to [Private Cloud Gateway](../../pcg/pcg.md). To learn how to connect a PCG to an
+  Azure cloud account, refer to the [Register and Manage Azure Cloud Account](./azure-cloud.md) guide.
+
+- A Private DNS Zone is required to use the Private API Server load balancer. You also need to ensure the vNETs used for
+  the cluster supports
+  [Private DNS resolving](https://learn.microsoft.com/en-us/azure/dns/dns-private-resolver-overview). us To learn more
```

Comment:
```suggestion
  [Private DNS resolving](https://learn.microsoft.com/en-us/azure/dns/dns-private-resolver-overview). To learn more
```
I have a question about the private DNS zone. In Kevin's screenshot, it says "Private DNS Zone (Optional)". Is it really mandatory in this case? :)